routers/api/v1/api.go

```diff
@@ -1196,6 +1196,7 @@ func Routes() *web.Router {
 						m.Patch("", bind(api.EditBranchProtectionOption{}), mustNotBeArchived, repo.EditBranchProtection)
 						m.Delete("", repo.DeleteBranchProtection)
 					})
+					m.Post("/priority/move", bind(api.UpdateBranchProtectionPriories{}), mustNotBeArchived, repo.UpdateBranchProtectionPriories)
```

Comment:
not sure if we should call it move or update ...

... i explizite did not make it just `/priority` so we can e.g.  add `/priority/reset`, `/priority/autosort`, ... if we want to later

Reply:
dffb634eff0afe7217688b1e4a86a17ef58615d9